test/over_react_test/custom_matchers_test.dart

```diff
@@ -527,9 +540,11 @@ main() {
               shouldPass(() => mount(Sample()()), hasLog('Just a lil warning', consoleConfig: warnConfig));
             });
 
-            test('simple usage with error config', () {
-              shouldPass(() => mount(Sample()()), hasLog('shouldNeverBeNull', consoleConfig: errorConfig));
-            });
+            if (runtimeSupportsPropTypeWarnings()) {
+              test('simple usage with error config', () {
+                shouldPass(() => mount(Sample()()), hasLog('shouldNeverBeNull', consoleConfig: errorConfig));
+              });
+            }
```

Comment:
#nit since this isn't directly the `propType` matcher, `Sample` could look like:
```dart
(Sample()
  ..shouldNeverBeNull = true
  ..shouldErrorInMount = true
)()
```

and then the matcher could look for that error instead and the test would be valid again!

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/5e736cd83aa9091f0e0dd699c9e3d8020ce7fd4f